<launch>

  <remap from="image_percept" to="worldmodel/image_percept" />

  <param name="use_sim_time" value="true" />

   <node pkg="tf" type="static_transform_publisher" name="base_to_kinect_tf" args="0.0 0 0.0 0 0 0 base_link /camera_link 100" />

    <include file ="$(find openni_launch)/launch/openni.launch" />

     <node pkg="QRcode_detection" type="QRcode_detection" name="QRcode_detection" output="screen" />
   
 <node pkg = "rqt_gui" type ="rqt_gui" name="rqt_gui" respawn="true">
</node>

  
</launch>
